libs/web/providers/src/CENNZExtensionProvider.tsx

```diff
@@ -39,6 +48,19 @@ export const CENNZExtensionProvider: FC<CENNZExtensionProviderProps> = ({
 	const [module, setModule] = useState<typeof Extension>();
 	const [accounts, setAccounts] = useState<Array<InjectedAccountWithMeta>>();
 
+	const promptDisableExtension = useCallback(() => {
+		const url =
+			"https://support.google.com/chrome_webstore/answer/2664769?p=enable_extensions&rd=1#extensionpermissions";
+
+		const confirmed = window.confirm(
```

Comment:
alert should be fine, we don't even need to send them to the Google Chrome help as it will get block anyway.